wdl-doc/src/docs_tree.rs

```diff
@@ -846,8 +844,9 @@ impl DocsTree {
                         li class="flex place-content-center" {
                             img x-show="search !== '' && searchedNodes.length === 0" src=(self.get_asset(base, "search.svg")) class="size-8" alt="Search icon";
                         }
-                        li class="flex place-content-center" {
-                            p x-show="search !== '' && searchedNodes.length === 0" class="" x-text="'No results found for \"' + search + '\"'" {}
+                        li x-show="search !== '' && searchedNodes.length === 0" class="flex gap-1 place-content-center text-center break-words whitespace-normal text-sm text-slate-500" {
+                            span x-text="'No results found for '" {}
+                            span x-text="` \"${search}\"`" class="text-slate-50" {}
```

Comment:
```suggestion
                            span x-text="`\"${search}\"`" class="text-slate-50" {}
                            span x-text="'" {}
```
I think there's a missing trailing comma. This should fix it